backend/de.metas.business/src/main/java/de/metas/order/impl/OrderBL.java

```diff
@@ -129,6 +130,8 @@
 	private static final String SYS_CONFIG_MAX_HADDEX_AGE_IN_MONTHS = "de.metas.order.MAX_HADDEX_AGE_IN_MONTHS";
 	private static final AdMessageKey MSG_HADDEX_CHECK_ERROR = AdMessageKey.of("de.metas.order.CustomerHaddexError");
 
+	public static final String SYSCONFIG_C_Invoice_PaymentRule = "de.metas.invoice.C_Invoice_PaymentRule";
```

Comment:
please don't introduce another method that is basically identical to IInvoiceBL.getDefaultPaymentRule(). instead, please use IInvoiceBL.getDefaultPaymentRule() like e.g. C_OrderLine_Handler does (or extract it into a service that sits near the `PaymentRule` class)